### PR TITLE
Add a pre-release warning

### PR DESCRIPTION
**Pull request by meisenzahl**
Closes https://github.com/elementary/onboarding/issues/110

### CODE REVIEW

**Comment by meisenzahl** [comment]
Screenshot von 2020-10-28 19 20 49

**Comment by meisenzahl** [comment]
@cassidyjames and @danrabbit I would love to get feedback.

Could you guide me in the right direction to get the alignment in the list right=

**Comment by cassidyjames** [comment]
@meisenzahl just a left-aligned bulleted list would make sense to me. :smile:

**Comment by meisenzahl** [comment]
@cassidyjames I meant how to implement such a list.

Screenshot von 2020-10-28 20 01 30

Little `Gtk.LinkButton` does not want to play with his friend the `Gtk.Label`'s...

**Comment by meisenzahl** [comment]
Bildschirmfoto von 2020-11-04 19 45 19

**Comment by cassidyjames** [comment]
@meisenzahl sorry I missed this yesterday, this week is crazy hard for focusing here in the US. :sweat::ballot_box::crossed_fingers: I will take a look today and play with some layout/copy options.

File: src/Views/PreReleaseWarningView.vala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Authored by: Marius Meisenzahl <mariusmeisenzahl@gmail.com>
+ */
+
+public class Onboarding.PreReleaseWarningView : AbstractOnboardingView {
+    public PreReleaseWarningView () {
+        Object (
+            view_name: "pre-release-warning",
+            description: _("You're running an early daily build of %s. Things will be broken.").printf (Utils.os_name),
```
Comment (cassidyjames) [inline src/Views/PreReleaseWarningView.vala:24]: More similar to the copy we use on the builds site:

```suggestion
            description: _("You're running a <b>pre-release</b> daily development build of %s. Things may be broken. Current major known issues include, but are not limited to:").printf (Utils.os_name),
```

File: src/Views/PreReleaseWarningView.vala [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            icon_name: "dialog-warning",
+            title: _("Pre-release version of %s").printf (Utils.os_name)
```
Comment (cassidyjames) [inline src/Views/PreReleaseWarningView.vala:26]: Simpler and more similar to builds site:

```suggestion
            title: _("Early Access Build")
```

File: src/Views/PreReleaseWarningView.vala [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+        var style_label = new Gtk.Label ("• " + _("The visual style is unfinished")) {
+            hexpand = true,
+            xalign = 0
+        };
+        custom_bin.add (style_label);
+
+        var more_link = new Gtk.LinkButton.with_label (
+            "https://github.com/orgs/elementary/projects/55",
+            _("More…")
```
Comment (cassidyjames) [inline src/Views/PreReleaseWarningView.vala:47]:
```suggestion
            _("View the 6.0 Release Project")
```

**Review by cassidyjames — APPROVED** [review]
Approving as-is and we can iterate. :)

**Pull request merged**